python/src/worker/tasks/dotplot.py

```diff
@@ -27,19 +26,35 @@ def _format_result(self, result):
         backoff.expo, requests.exceptions.RequestException, max_time=30
     )
     def compute(self):
-        request = {"nGenes": self.task_def["nGenes"], "type":self.task_def["type"], "config":self.task_def["config"]}
-        
-        if self.task_def["type"] is "custom":
-            request["genes"] = self.task_def["genes"]
-        else:
-            request["nGenes"] = self.task_def["nGenes"]
 
-        typeOfSets = self.task_def["typeOfSets"]
+        input = self.task_def["input"]
+        subset = self.task_def["subset"]
 
-        cellSets = get_cell_sets(self.experiment_id)
+        request = {"markerGenes": self.task_def["markerGenes"]}
```

Comment:
Change this to a name that makes it clearer that this is a bool

Reply:
this is changed to `useMarkerGenes`